DEVELOPMENT.md

```diff
@@ -82,6 +82,21 @@ $ wget -q https://boostorg.jfrog.io/artifactory/main/release/1.65.1/source/boost
 
 ```
 
+- CentOS 8
+
+```shell
+# Enable powertools
+$ sudo dnf config-manager --set-enabled powertools
```

Comment:
You may need to install config-manager first, or check whether config-manager is installed.

Reply:
Correct, in some minimal Installation, `dnf-plugins-core` may missing.